cmd/okctl/hooks/kubeconfig.go

```diff
@@ -0,0 +1,26 @@
+package hooks
+
+import (
+	"fmt"
+
+	"github.com/oslokommune/okctl/pkg/okctl"
+	"github.com/spf13/cobra"
+)
+
+// WriteKubeConfig writes the Kubernetes configuration to disk
+func WriteKubeConfig(o *okctl.Okctl) RunEer {
```

Comment:
```suggestion
func EnsureKubeConfig(o *okctl.Okctl) RunEer {
```
Maybe?

Reply:
Hmm, naming :smile: 

I used WriteKubeConfig because that's exactly what it does. I could and perhaps even have been more specific, WriteKubeConfigToDisk.

EnsureKubeConfig is more vague, which is not bad, but for me it communicates something else, namely that "make sure the Kubeconfig exists in whatever way we need, but we don't care about which way".

When it comes to Kubeconfig, this is a file that must exist on disk, so I'm still leaning against `WriteKubeConfig`.

Reply:
Sure! I was thinking from the perspective of the user (developer, command); as a command, I want to make sure the kubeconfig exists. Doesn't really matter. You do you